react/features/toolbox/components/web/Toolbox.js

```diff
@@ -996,6 +1004,10 @@ class Toolbox extends Component<Props, State> {
                 key = 'settings'
                 showLabel = { true }
                 visible = { this._shouldShowButton('settings') } />,
+            <MuteEveryoneButton
+                key = 'mute-everyone'
+                showLabel = { true }
+                visible = { _isModerator } />,
```

Comment:
Also I'm wondering if the `_isModerator` flag should be moved to `MuteEveryoneButton`'s `mapStateToProps`?

Reply:
Makes sense indeed!

Reply:
I looked into this, it seems this toolbox item is the only one that needs to be visible if you are a moderator. There is quite a complicated inheritance chain behind all toolbox items - do you have any suggestion on how to move this visibility flag into `MuteEveryoneButton` without needing to alter a lot of other components along and across the inheritance chain?

Reply:
I think a pattern like this would work: https://github.com/jitsi/jitsi-meet/blob/98698ba89accac8fa707095cbc5020199012686c/react/features/etherpad/components/SharedDocumentButton.js#L73

Reply:
thank you @saghul , I missed that :( updated